src/services/ledger/index.ts

```diff
@@ -79,6 +79,19 @@ export const LedgerService = (): ILedgerService => {
       return new UnknownLedgerError(err)
     }
   }
+
+  const isLnTxRecorded = async (
+    paymentHash: PaymentHash,
+  ): Promise<boolean | LedgerServiceError> => {
+    try {
+      const result = await Transaction.countDocuments({
+        hash: paymentHash,
```

Comment:
do we have to check with `pending: false`? 🤔

Reply:
Also, do we need to check that the journal entry isn't `voided`? Not sure if any other parts of our code take that into account

Reply:
I see what you guys mean, but see some issues - lets discuss it at standup.